frontend/integration-tests-cypress/cypress/support/utils.ts

```diff
@@ -0,0 +1,13 @@
+export enum timeouts {
```

Comment:
I'd be inclined to do a directory here again:

```ts
/support/utils/timeouts.ts
/support/utils/selectors.ts
/support/utils/utils.ts // for the `testName` and things that actually don't fit a category
```

Reply:
fixed